README.md

```diff
@@ -78,5 +76,6 @@ THe following is a good summary of available command-line options within the scr
 
 This section is a WIP. This section will be devoted to understanding how the data that was inserted into the `theme.json` can be used in the editor.
 
-[png-pro-plugin-usage]: /docs/assets/pro-plugin-usage.png
-[png-non-pro-plugin-usage]: /docs/assets/non-pro-plugin-usage.png
\ No newline at end of file
+[png-pro-plugin-usage]: https://github.com/Automattic/vip-design-system-bridge/blob/trunk/docs/assets/pro-plugin-usage.png
```

Comment:
@ingeniumed Updated all of these links to use the full path to images. This is a bit more brittle, but it's a fool-proof way to ensure if the README is embedded in an npm package or elsewhere that relative URLs don't break.